### PR TITLE
[JBPM-8497] Document integration with production databases

### PR DESCRIPTION
**Pull request by ruromero**
Signed-off-by: Ruben Romero Montes <rromerom@redhat.com>

Fix https://issues.jboss.org/browse/JBPM-8497

### CODE REVIEW

File: process-migration-service/src/main/resources/META-INF/persistence.xml [hunk 1]
```diff
@@ -1,18 +1,17 @@
...
-	xsi:schemaLocation="http://xmlns.jcp.org/xml/ns/persistence http://xmlns.jcp.org/xml/ns/persistence/persistence_2_1.xsd">
-	<persistence-unit name="migration-unit"
-		transaction-type="JTA">
-		<provider>org.hibernate.jpa.HibernatePersistenceProvider</provider>
-		<class>org.kie.processmigration.model.Migration</class>
-		<class>org.kie.processmigration.model.Plan</class>
-		<class>org.kie.processmigration.model.MigrationReport</class>
-		<properties>
-			<property name="hibernate.max_fetch_depth" value="3" />
-			<property name="hibernate.hbm2ddl.auto" value="create" />
```
Comment (MarianMacik) [inline process-migration-service/src/main/resources/META-INF/persistence.xml]: This part has been omitted. So now Hibernate will not create database tables for PIM schema. I guess we should leave create here for now unless we want to create DDL scripts for each database.

File: process-migration-service/README.md [hunk 2]
```diff
@@ -22,46 +22,46 @@ Is the execution of a defined plan, applied to a set of process instances. These
...
 
 * JRE 1.8 or greater
 * Running KIE Process server
 
-# Build
+## Build
 
-```
-$ mvn clean package
+```{bash}
```
Comment (MarianMacik) [inline process-migration-service/README.md]: One note. According to GitHub Markdown, this should be like ` ```bash` for syntax highlighting.
https://help.github.com/en/articles/creating-and-highlighting-code-blocks#syntax-highlighting

File: process-migration-service/src/main/resources/META-INF/persistence.xml (hunk 1 above)
Comment (ruromero) [reply process-migration-service/src/main/resources/META-INF/persistence.xml]: If we leave `create` each time the application starts, the database schema and data will be dropped. What do you suggest?

**Comment by MarianMacik** [comment]
Well, according to the [Hibernate documentation](https://docs.jboss.org/hibernate/orm/5.4/userguide/html_single/Hibernate_User_Guide.html#configurations-hbmddl), `update` could be the way to go. This won't drop the schema.

**Pull request merged**